supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesCacheHitCopyingActor.scala

```diff
@@ -26,46 +31,64 @@ case class JesCacheHitCopyingActor(override val jobDescriptor: BackendJobDescrip
                                    serviceRegistryActor: ActorRef) extends BackendCacheHitCopyingActor {
 
   override val configurationDescriptor = jesConfiguration.configurationDescriptor
+  private lazy val completionPromise = Promise[BackendJobExecutionResponse]()
```

Comment:
The async backend actors use it to derive a `Future` which can be completed by the completion of this `Promise`, which may or may not fit the shape of the problem here.